x-pack/plugins/maps/server/mvt/get_grid_tile.ts

```diff
@@ -5,13 +5,16 @@
  * 2.0.
  */
 
-import { Logger } from 'src/core/server';
+import { CoreSetup, Logger } from 'src/core/server';
```

Comment:
setup methods should not be used outside of plugin.setup method and only for registration. If you can not use context.core, how about passing in CoreStart?